RestaurentManagement/src/main/java/com/restapi/controllers/MenuController.java

```diff
@@ -78,4 +78,26 @@ public ResponseEntity<SuccessResponse> deleteMenu(@PathVariable UUID id) {
                 "No data response");
         return new ResponseEntity<>(successResponse,HttpStatus.OK);
     }
+
+    @GetMapping("/search")
+    public ResponseEntity<SuccessResponse> searchMenus(@RequestParam(required = false) String keyword) {
+        SuccessResponse successResponseNotFound = new SuccessResponse(
+                HttpStatus.OK.value(),
+                HttpStatus.OK.getReasonPhrase(),
+                "Search menus successfully!",
+                "No data found!");
+        if(keyword==null)
+            return new ResponseEntity<>(successResponseNotFound,HttpStatus.OK);
```

Comment:
throw exception in service class

Reply:
Pls research again. The HTTP status to use for successful API and exception API.